api/src/main/java/io/strimzi/api/kafka/Crds.java

```diff
@@ -43,6 +43,7 @@
 public class Crds {
 
     public static final String CRD_KIND = "CustomResourceDefinition";
+    public static final String CRD_CATEGORY = "strimzi";
```

Comment:
I'd call this `STRIMZI_CATEGORY`. I'm also not sure this is really the correct class to declare it on. Though I can't think of a better one. Looking at our CRD-annotated classes, such as `Kafka`, we have various constants which are common across numerous classes:

public static final String CRD_API_VERSION = "apiextensions.k8s.io/v1beta1";
public static final String RESOURCE_GROUP = "kafka.strimzi.io";

But I'm not sure even that's enough to warrant some `CrdConstants` class. Thoughts @scholzj ?

Reply:
Tested with both `oc` and `kubectl`

Reply:
Imho maybe we could have a CRD related constants class but if we do it in this PR we'll have a lot of unrelated changes.

Reply:
I've opened an issue for this: https://github.com/strimzi/strimzi-kafka-operator/issues/2437